packages/components/src/card/card-footer/index.js

```diff
@@ -15,6 +15,9 @@ export const defaultProps = {
 	size: 'medium',
 };
 
+/**
+ * @param { import('../types').FooterProps & { className: string } } props
```

Comment:
Or we might as well add `className` to the `FooterProps` type?

Reply:
Resolving in favour of the other comments about Header/Footer props